packages/framework/src/Model/Order/Item/OrderItemDataFactory.php

```diff
@@ -4,6 +4,20 @@
 
 class OrderItemDataFactory implements OrderItemDataFactoryInterface
 {
+    /**
+     * @var \Shopsys\FrameworkBundle\Model\Order\Item\OrderItemPriceCalculation|null
+     */
+    protected $orderItemPriceCalculation;
+
+    /**
+     * @required
```

Comment:
please add note that this will be changed to constructor injection in next major version (already done in the SSFW)

Reply:
I don't think we ever decided that we want to change *all* setter injections to constructon injections in the next (or every) major release. But here it makes sense I guess, added in 15dc0a240.

Reply:
Depends what do you imagine by "we ever decided". If you think we had a whole team meeting and we've written down conclusion, then no. If you think couple of developers (3: PIA, DEV, CR) agreed on this, then yes :bird: